exercises/forth/example.py

```diff
@@ -12,7 +12,7 @@ def is_integer(string):
 
 def evaluate(input_data):
     defines = {}
-    while input_data[0][0] == ':':
+    while input_data[0][:1] == ':':
```

Comment:
Took me a moment to understand how this changed anything... good catch! I'm not sure I would've thought of that, probably would've done something clunky, such as:
```Python
while input_data[0] != '' and input_data[0][0] == ':':
...
```

Reply:
Yeah, it's a bit of a weird string-slicing trick :stuck_out_tongue: